### ISSUE TITLE
[BUG] Informativity GUI does not correctly select digraphs

### ISSUE THREAD

**Issue by zl-zhou** [PhonologicalCorpusTools/CorpusTools#807]
**Describe the bug**
When using the windows that comes up with "Select one or more segments" on the informativity window, selection of any segment represented by more than one letter causes only the first letter to be entered into the list of segments

**Sample corpus file**
Behavior can be seen in iphon_with_homographs

**To Reproduce**
1. Go to "Analysis"
2. Click on "Calculate informativity..."
3. Click on "Select one or more segments from the inventory"
4. Select "NG, SH, AE"
5. Click "Add"
6. Only "A", "N", and "S" are added to the list

**Operating system and PCT version**
 - OS: Windows 10 Pro Version 10.0.19043 Build 19043
 - PCT version: 1.5.1

**Additional context**
To make this work, I've been adding all segments to the inventory and then individually removing segments.

**Comment by stannam** [comment]
Hi @zl-zhou. Thanks for letting us know about this issue. 

It seems like there is an error in the existing function for adding a segment. For some reason, it mistakenly forces all sounds to be in a single symbol. This error was in the GUI for the Informativity analysis, meaning that other functions, including the informativity analysis itself, are not affected.

The problem is that ARPABET, the transcription system that IPHOD uses, has NG, SH, AE for ŋ ʃ æ. So, for example, NG, SH, AE are forced to be N, S, A. And N, S, A, not the original segments, are delivered to the analysis function!

Therefore, I have simply removed the code snippet that forces it. I confirmed that Informativity works correctly now in the latest codes.

So.. I temporarily created the link below for downloading an executable. This version should be free of this issue :)
https://www.dropbox.com/s/8jq83tl0fft3xt4/Phonological%20CorpusTools_dev.exe?dl=1
